### DISCUSSION TITLE
Booster pack commands

### DISCUSSION THREAD

**Discussion by HenkerX64** [chr233/ASFEnhance#41] [功能建议 / Ideas]
Hello,

it will be nice to have booster pack creation command.

### Example:
|  Command | Shorthand | Access   | Description |
| ------------ | ------------ | ---------- | ---------- |
| `CREATEBOOSTERPACK [Bots] <AppIDs>` |  `CBP` | `Operator `| Create booster pack |
| `BOOSTERPACKCATALOG [Bots] <AppIDs>` |  -  | `FamilySharing` | Get bot's available booster pack list |

Example from node project:
https://github.com/DoctorMcKay/node-steamcommunity/blob/master/components/market.js#L224
https://github.com/DoctorMcKay/node-steamcommunity/blob/master/components/market.js#L155

**Comment by woctezuma** [comment]
For info, there are already [plugins](https://github.com/Citrinate/BoosterManager) which cover this subject.

**Reply by HenkerX64** [reply]
Thanks, seems to be new and interesting plugin, and it is only for **ASF-generic** version, like [the plugin from Ryzhehvost](https://github.com/Ryzhehvost/BoosterCreator).

**Comment by chr233** [comment]
maybe I will add this function after current sale event ends.
